MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -44,95 +66,37 @@ public class DistanceFormatter: LengthFormatter {
      The user’s `Locale` is used here to set the units.
     */
     public func string(from distance: CLLocationDistance) -> String {
-        let miles = distance / metersPerMile
-        let feet = miles * feetPerMile
-        
         // British roads are measured in miles, yards, and feet. Simulate this idiosyncrasy using the U.S. locale.
-        let isBritish = Locale.current.identifier == "en-GB"
-        numberFormatter.locale = isBritish ? Locale(identifier: "en-US") : Locale.current
-        
-        var unit: LengthFormatter.Unit = .millimeter
-        unitString(fromMeters: distance, usedUnit: &unit)
-        let replacesYardsWithMiles = unit == .yard && miles > 0.2
-        let showsMixedFraction = (unit == .mile && miles < 10) || replacesYardsWithMiles
-        
-        // An elaborate hack to use vulgar fractions with miles regardless of
-        // language.
-        if showsMixedFraction {
-            numberFormatter.positivePrefix = "|"
-            numberFormatter.positiveSuffix = "|"
-            numberFormatter.decimalSeparator = "!"
-            numberFormatter.alwaysShowsDecimalSeparator = true
-        } else {
-            numberFormatter.positivePrefix = ""
-            numberFormatter.positiveSuffix = ""
-            numberFormatter.decimalSeparator = nonFractionalLengthFormatter.numberFormatter.decimalSeparator
-            numberFormatter.alwaysShowsDecimalSeparator = nonFractionalLengthFormatter.numberFormatter.alwaysShowsDecimalSeparator
+        let localeIdentifier = numberFormatter.locale.identifier
+        if localeIdentifier == "en-GB" || localeIdentifier == "en_GB" {
+            numberFormatter.locale = Locale(identifier: "en-US")
         }
         
-        if approx && !showsMixedFraction {
-            numberFormatter.usesSignificantDigits = true
-            numberFormatter.maximumSignificantDigits = 2
-        } else {
-            numberFormatter.usesSignificantDigits = false
-            numberFormatter.maximumFractionDigits = showsMixedFraction ? 2 : 0
-        }
+        numberFormatter.positivePrefix = ""
+        numberFormatter.positiveSuffix = ""
+        numberFormatter.decimalSeparator = nonFractionalLengthFormatter.numberFormatter.decimalSeparator
+        numberFormatter.alwaysShowsDecimalSeparator = nonFractionalLengthFormatter.numberFormatter.alwaysShowsDecimalSeparator
         numberFormatter.roundingIncrement = 0.25
```

Comment:
Is there a reason to leave the default rounding increment at 0.25 if we're truncating to 1 decimal place?